src/test/javascript/spec/component/course/course-exercises.component.spec.ts

```diff
@@ -103,28 +96,28 @@ describe('CourseExercisesComponent', () => {
     });
 
     afterEach(() => {
-        sinon.restore();
-        jest.clearAllMocks();
+        jest.restoreAllMocks();
     });
 
     it('should initialize', () => {
-        expect(component.course).to.deep.equal(course);
-        expect(courseCalculationSpy.mock.calls.length).to.equal(2);
-        expect(courseCalculationSpy.mock.calls[0][0]).to.eq(course.id);
-        expect(courseCalculationSpy.mock.calls[1][0]).to.eq(course.id);
+        expect(component.course).toEqual(course);
+        expect(courseCalculationSpy.mock.calls.length).toEqual(2);
+        expect(courseCalculationSpy.mock.calls[0][0]).toEqual(course.id);
+        expect(courseCalculationSpy.mock.calls[1][0]).toEqual(course.id);
     });
 
     it('should invoke setSortingAttribute', () => {
         const sortingButton = fixture.debugElement.query(By.css('#dueDateSorting'));
-        expect(sortingButton).to.exist;
+        expect(sortingButton).not.toBe(null);
+
         sortingButton.nativeElement.click();
-        expect(component.sortingAttribute).to.equal(SortingAttribute.DUE_DATE);
+        expect(component.sortingAttribute).toEqual(SortingAttribute.DUE_DATE);
```

Comment:
I think enums should be checked with `toBe()` as well. Try if it works
```suggestion
        expect(component.sortingAttribute).toBe(SortingAttribute.DUE_DATE);
```